app/views/galleries/index.html.erb

```diff
@@ -12,7 +12,7 @@
         <% @galleries.each do |gallery| %>
         <li>
             <%= link_to gallery.name, gallery_path(gallery.id) %> - 
-            <%= gallery.description %>
+            <%= link_to gallery.user[:email], gallery.user %>
```

Comment:
You can access the `email` attribute either like you are (like a hash), or by a method call `gallery.user.name`. The preferred form is to use the method call.